File: BookWorm-NET/BookWormNET/dto/ProductBeneficiarydto.cs
namespace BookWormNET.dto
{
    public class ProductBeneficiarydto
    {
        public int ProdbenId { get; set; }
        public string BeneficiaryName { get; set; }
        public string ProductName { get; set; }

        public decimal RoyaltyReceived { get; set; }
        public decimal? RoyaltyPercent { get; set; }
        public decimal? TotalRoyalty { get; set; }
    }
}

File: BOOKWORM_PRO/Bookworm/src/main/java/com/example/dto/ProductBeneficiaryDTO.java
package com.example.dto;

import java.math.BigDecimal;

public class ProductBeneficiaryDTO {

	private int prodbenId;
    private String beneficiaryName;
    private String productName;
    private BigDecimal royaltyReceived;
	public int getProdbenId() {
		return prodbenId;
	}
	public void setProdbenId(int prodbenId) {
		this.prodbenId = prodbenId;
	}
	public String getBeneficiaryName() {
		return beneficiaryName;
	}
	public void setBeneficiaryName(String beneficiaryName) {
		this.beneficiaryName = beneficiaryName;
	}
	public String getProductName() {
		return productName;
	}
	public void setProductName(String productName) {
		this.productName = productName;
	}
	public BigDecimal getRoyaltyReceived() {
		return royaltyReceived;
	}
	public void setRoyaltyReceived(BigDecimal royaltyReceived) {
		this.royaltyReceived = royaltyReceived;
	}
    
    
    
}
